app/controllers/idv/how_to_verify_controller.rb

```diff
@@ -38,7 +38,7 @@ def self.step_info
         controller: controller_name,
         next_steps: [:hybrid_handoff, :document_capture],
         preconditions: ->(idv_session:, user:) do
-          self.enabled?
+          self.enabled? && idv_session.idv_consent_given
         end,
         undo_step: ->(idv_session:, user:) {}, # clear any saved data
```

Comment:
This PR looks good, and you could go ahead and add idv_session.skip_doc_auth = nil for undo_step.

Reply:
@svalexander  I will take care of this. In [PR # 9598](https://github.com/18F/identity-idp/pull/9598/files), I am setting the value so I think it makes sense to unset it in the same PR.